src/lib-sana/Utils/Utils.cpp

```diff
@@ -1,5 +1,52 @@
+#include <random>
+#include <sstream>
+#include <fstream>
+#include <iterator>
+#include "Random.hpp"
 #include "Utils.hpp"
 
 using namespace std;
 
+RandomNumberGenerator randObj;
```

Comment:
We want to keep track of seed (most likely in SANAConfig) and sometimes we might want to set the seed direcly, so better not use `static` functions for generating the random numbers.